kodi_addon_checker/check_dependencies.py

```diff
@@ -31,9 +31,14 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml, bran
         :branch_name: name of the kodi branch/version
     """
 
-    deps = _get_users_dependencies(parsed_xml)
+    optional_deps, deps = _get_users_dependencies(parsed_xml)
     ignore = _get_ignore_list(branch_name)
 
+    if optional_deps:
+        for dependency, version in optional_deps.items():
+            report.add(Record(INFORMATION, "%s with version %s is set as an optional dependency" %
```

Comment:
Again, this does not add any value. The issue (INFO) in this case should only be reported if the dependency is not available in the repository. See the use case of the issue you're trying to solve. 

Setting dependencies as optional is per-si is an option for any addon...